src/renderers/__tests__/refs-test.js

```diff
@@ -16,78 +16,80 @@ var ReactDOMFeatureFlags = require('ReactDOMFeatureFlags');
 var ReactTestUtils = require('ReactTestUtils');
 
 /**
- * Counts clicks and has a renders an item for each click. Each item rendered
- * has a ref of the form "clickLogN".
+ * Render a TestRefsComponent and ensure that the main refs are wired up.
+ * we also define the classes in this function to ensure that they are
+ * wired up properly in accordance to how Jest might auto mock modules
```

Comment:
Hm, could you explain how this is related to automocking?
If you mean resetting modules, it's a different feature from automocking.
I don't think we have automocking enabled in React repo.